src/apis/stockx/user/helpers.js

```diff
@@ -0,0 +1,134 @@
+import cheerio from 'cheerio';
```

Comment:
Look into alternatives for cheerio. If this library will be included in projects, we want to minimize it (bundle size) as much as possible. Cheerio is unnecessarily large if we just need to grab a few form values in one file imho.

Reply:
Maybe [hyntax](https://github.com/mykolaharmash/hyntax)? It's a total of 4.7kB compared to cheerio which is 319.3kB (when minified).

Reply:
https://www.npmjs.com/package/dom-parser looks like a good option.

Reply:
Perfect! Even smaller 🙂

Reply:
Implemented [html-dom-parser](https://www.npmjs.com/package/html-dom-parser) which is 3.3kB in size. Resolving...